po/sympa/sympa.pot

```diff
@@ -6876,6 +6884,10 @@ msgstr ""
 msgid "Request new password"
 msgstr ""
 
+#: default/web_tt2/report_abuse.tt2:3
+msgid "Report abuse"
+msgstr ""
+
```

Comment:
`*.pot` file need not updating: It will be done during release process.

Reply:
I didn't knew that, sorry.

Reply:
Don't worry. I'm glad now you know about common job.

Reply:
Removed from the commit.